packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -133,8 +133,10 @@
 import {developerDashboardFqdn} from '@shopify/cli-kit/node/context/fqdn'
 import {webhooksRequest} from '@shopify/cli-kit/node/api/webhooks'
 import {functionsRequestDoc} from '@shopify/cli-kit/node/api/functions'
+import {fileExists, readFile} from '@shopify/cli-kit/node/fs'
 
 const TEMPLATE_JSON_URL = 'https://cdn.shopify.com/static/cli/extensions/templates.json'
+const TEMPLATE_PATH_ENV_VARIABLE = 'SHOPIFY_APP_TEMPLATES_JSON_PATH'
```

Comment:
Let's put this in `app/src/cli/constants.ts` -- there's a handful of other app-specific environment variables there.

Reply:
I debated about this point - it's kind of a little debugging hack, but we do have some debugging hacks there as well!